<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>603</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextEdit" name="xLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>230</width>
     <height>110</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="xtest_btn">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>80</y>
     <width>230</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>X Test</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="yLabel_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>290</y>
     <width>230</width>
     <height>130</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="xLabel_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>120</y>
     <width>230</width>
     <height>110</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="exit_btn">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>440</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>실행</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ytest_btn">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>250</y>
     <width>230</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Y Test</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ytrain_btn">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>250</y>
     <width>230</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Y Train</string>
   </property>
  </widget>
  <widget class="QPushButton" name="xtrain_btn">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>230</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>X Train</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="yLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>290</y>
     <width>230</width>
     <height>130</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <signal>signal1()</signal>
  <signal>select()</signal>
  <slot>clickedx()</slot>
  <slot>clickedy()</slot>
 </slots>
</ui>
